src/serverConfig.ts

```diff
@@ -12,37 +12,28 @@ async function getVSCodeProductJson() {
     return vscodeProductJson;
 }
 
-let vscodePackageJson: any;
-async function getVSCodePackageJson() {
-    if (!vscodePackageJson) {
-        const packageJsonStr = await fs.promises.readFile(path.join(vscode.env.appRoot, 'package.json'), 'utf8');
-        vscodePackageJson = JSON.parse(packageJsonStr);
-    }
-
-    return vscodePackageJson;
-}
-
 export interface IServerConfig {
     version: string;
     commit: string;
     quality: string;
-    release?: string; // vscodium specific
+    release?: string; // vscodium-like specific
     serverApplicationName: string;
     serverDataFolderName: string;
+    serverDownloadUrlTemplate?: string; // vscodium-like specific
 }
 
 export async function getVSCodeServerConfig(): Promise<IServerConfig> {
     const productJson = await getVSCodeProductJson();
-    const packageJson = await getVSCodePackageJson();
 
     const customServerBinaryName = vscode.workspace.getConfiguration('remote.SSH.experimental').get<string>('serverBinaryName', '');
 
     return {
         version: vscode.version.replace('-insider',''),
         commit: productJson.commit,
         quality: productJson.quality,
-        release: packageJson.release,
+        release: productJson.release,
```

Comment:
The release number is also available in the `product.json` so there is no need to load `package.json`.